package.json

```diff
@@ -21,6 +21,7 @@
     "mariadb": "2.4.1",
     "multer": "1.4.2",
     "sequelize": "6.2.4",
+    "sequelize-cli": "6.2.0",
```

Comment:
I don't think this should be in here. It's needed as a global dependency but not here. Please run `npm uninstall sequelize-cli`